api/data/tacoma_db.go

```diff
@@ -2,43 +2,63 @@ package data
 
 import (
 	"context"
+	"fmt"
+	"time"
 
 	"github.com/gobuffalo/nulls"
 	"github.com/jackc/pgx/v4"
 )
 
 // TacomaOfficer is the object model for Tacoma PD officers
 type TacomaOfficer struct {
-	FirstName  string       `json:"first_name,omitempty"`
-	LastName   string       `json:"last_name,omitempty"`
-	Title      string       `json:"title,omitempty"`
-	Department string       `json:"department,omitempty"`
-	Salary     nulls.String `json:"salary,omitempty"`
+	Date		time.Time		`json:"date,omitempty"`
+	FirstName	string			`json:"first_name,omitempty"`
+	LastName	string			`json:"last_name,omitempty"`
+	Title		string			`json:"title,omitempty"`
+	Department	string			`json:"department,omitempty"`
+	Salary		nulls.String	`json:"salary,omitempty"`
 }
 
-// TacomaOfficerMetadata retreives metadata describing the TacomaOfficer struct
-func (c *Client) TacomaOfficerMetadata() []map[string]string {
-	return []map[string]string{
-		{
+// TacomaOfficerMetadata retrieves metadata describing the TacomaOfficer struct
+func (c *Client) TacomaOfficerMetadata() DepartmentMetadata {
+	var date time.Time
+	err := c.pool.QueryRow(context.Background(),
+		`
+			SELECT date
+			FROM tacoma_officers
+			WHERE date = (select max(date) from tacoma_officers);
```

Comment:
Same comment here regarding subquerying
```suggestion
			SELECT max(date) as date
			FROM tacoma_officers
```